ccpm/scripts/setup-env-from-k8s.sh

```
@@ -0,0 +1,212 @@
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# Use current working directory as project directory (not script's parent)
+PROJECT_DIR="${PROJECT_DIR:-$(pwd)}"
+
+echo "🔧 Setting up .env from Kubernetes secrets..."
+echo "   Namespace: $NAMESPACE"
+echo "   Target: $PROJECT_DIR/$ENV_FILE"
+
+# Backup .env before making changes
+backup_env() {
+    if [ -f "$PROJECT_DIR/$ENV_FILE" ]; then
+        local backup_file="$PROJECT_DIR/$ENV_FILE.backup.$(date +%Y%m%d_%H%M%S)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)